### ISSUE TITLE
Values for impatient-showdown-flavor

### ISSUE THREAD

**Issue by buhtz** [jcs-elpa/impatient-showdown#6]
From the docu it is unclear which values are possible for `impatient-showdown-flavor`.

**Comment by jcs090218** [comment]
Yeah... sorry about the documentation. The value can be found in the code:

https://github.com/jcs-elpa/impatient-showdown/blob/15927eb364d0a1cdaa0b05f82024444d21249fb0/impatient-showdown.el#L69-L74

These are flavors from showdown, see https://github.com/showdownjs/showdown#flavors.

**Comment by buhtz** [comment]
I assume I do something wrong with the syntax.

```
(use-package impatient-showdown
  :hook
  (markdown-mode . impatient-showdown-mode)
  :init
  (setq impatient-showdown-flavor "github")
  (setq impatient-showdown-background-color "#9d9d9d")
  (setq impatient-showdown-markdown-background-color "#f7f1ec")
  )
```

The result looks like an error

![grafik](https://user-images.githubusercontent.com/11861496/210820042-8e239ee3-943f-4dd3-bf1b-43f57191553c.png)

**Comment by jcs090218** [comment]
It should be a symbol, not a string:

```elisp
(setq impatient-showdown-flavor 'github)
```

**Comment by buhtz** [comment]
Ah. OK, now it "works" without errors.
But the visual result looks identical between each of the three options (default, vanilla and github).
